node/_tools/test/parallel/test-event-emitter-method-names.js

```diff
@@ -38,5 +38,9 @@ assert.strictEqual(E.off, E.removeListener);  // Same method.
 Object.getOwnPropertyNames(E).forEach(function(name) {
   if (name === 'constructor' || name === 'on' || name === 'off') return;
   if (typeof E[name] !== 'function') return;
+  if (name === 'listenerCount') {
+    assert.strictEqual(E[name].name, '_listenerCount');
+    return;
+  };
```

Comment:
We want to avoid changing this file because this file is automatically copied from node.js code base.

Can we make `E[name].name` 'listenerCount' here?